finality-aleph/src/new_network/data_network/mod.rs

```diff
@@ -0,0 +1,54 @@
+use crate::{
+    crypto::Signature,
+    new_network::{connection_manager::AuthData, PeerId},
+    Decode, Encode, NodeIndex, SessionId,
+};
+use codec::Codec;
+
+pub mod aleph_network;
+mod rmc_network;
+mod split;
```

Comment:
None of the code below this line should be here, there are already new equivalents of all that in various other `mod.rs` files.

Reply:
Done